<launch>
    <group ns="pandora_vision_landoltc">
      
      <param name="camera_name" value="kinect" />
		
			<node name="landoltc_node" type="landoltc_node" pkg="pandora_vision_landoltc" respawn="False" output="screen" />
			<!-- Add in upper line to enable debugging: launch-prefix="kdbg" /-->

      <param name="patternPath" type="str" value="$(find pandora_vision_landoltc)/bold.jpg"/>  
       
    </group>
</launch>
